BaseApiBundle/DependencyInjection/OpenOrchestraBaseApiExtension.php

```diff
@@ -37,5 +37,11 @@ public function load(array $config, ContainerBuilder $container)
         $loader->load('transformer.yml');
         $loader->load('oauth2.yml');
         $loader->load('security.yml');
+
+        $filterPagincationConfig = __DIR__.'/../../../../open-orchestra-libs/OpenOrchestra/Pagination';
```

Comment:
why did you put that into the base-api-bundle when you should have put it in the base-bundle